ibis/expr/types/core.py

```diff
@@ -46,6 +46,18 @@ class Expr(Immutable, Coercible):
     __slots__ = ("_arg",)
     _arg: ops.Node
 
+    def __rich_console__(self, console, options):
+        import ibis
+
+        if not ibis.options.interactive:
```

Comment:
`options` is already imported at the top

Reply:
Done.